### ISSUE TITLE
[no-identical-title] issue with describe.each

### ISSUE THREAD

**Issue by idan-at** [jest-community/eslint-plugin-jest#786]
Hi,

This code will trigger a lint error:
```js
describe("my tests", () => {
  describe.each`
    description
    ${'a'}
  `("$description", () => {})

  describe.each`
    description
    ${'b'}
  `("$description", () => {})
});
```

because the `$description` string repeats on both.

I'd expect those cases to actually resolve the title to what will be used on runtime, and then search for duplications.

so for example in this case, we'll have [`a`, `b`] as [describeTitles](https://github.com/jest-community/eslint-plugin-jest/blob/main/src/rules/no-identical-title.ts#L15), and not `["$description", "$description"]`

**Comment by G-Rath** [comment]
> I'd expect those cases to actually resolve the title to what will be used on runtime

I don't think that's really feasible as there's a lot of ways you could pass data into `.each`, and an equally large number of ways you can do substitution.

I think for now we probably just shouldn't check duplicate titles for `.each`.
